prac_05/emails.py

```diff
@@ -0,0 +1,14 @@
+name_to_email = {}
+email = input("Email: ")
+while email:
+    names = email.split("@")[0].split(".")
+    name = " ".join([item.title() for item in names])
```

Comment:
Would this be easy to understand years later? Do you need a comment explaining what it does?